src/index.html

```diff
@@ -7,8 +7,46 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo" href="#home"></a>
+      <nav class="header__navigation">
+        <ul class="header__navlist">
+          <li class="header__listitem">
+            <a href="#apple" class="header__link is-active">Apple</a>
+          </li>
+          <li class="header__listitem">
+            <a href="#samsung" class="header__link">Samsung</a>
+          </li>
+          <li class="header__listitem">
+            <a href="#smartphones" class="header__link">Smartphones</a>
+          </li>
+          <li class="header__listitem">
+            <a href="#l&c"
+            class="header__link"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
please check code style for html in your about indentations [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) (13 point)

Reply:
Thank you for support, however i didn`t get what exactly isn`t fit on different widths. All sizes are the same as in a mockup and all test are passed. You want me not to use justify-content but margins?